.github/workflows/release.yml

```diff
@@ -23,27 +20,27 @@ jobs:
 
       - name: Setup git env
         run: |
-          git config --global user.email "ckipp@pm.me"
-          git config --global user.name "coc-metals CI"
+          git config --global user.email "bot@scalameta.org"
+          git config --global user.name "Scalameta bot"
 
       - name: Checkout
         uses: actions/checkout@v1
```

Comment:
while you're at it, you can maybe update checkout to v2. From what I see in the changelog, it should be possible to just `git push` instead of having a custom action (but it doesn't really matter)

Reply:
ooo, I'll look at that

Reply:
That's great. I was able to verify that this works [here](https://github.com/ckipp01/actions-sandbox/runs/427437512?check_suite_focus=true)

Reply:
Nice! Thanks for confirming, I'll make the change in the other projects as well

Reply:
I have a "metals" day at work today, so I can go through and update those as well.